docs/doc/15-sql-functions/131-geo-h3-functions/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: 'Geography H3 Functions'
```

Comment:
We already have a GEO doc:https://databend.rs/doc/sql-functions/geo-functions/

It seems no need add a new one.

Reply:
Fixed.